src/main/resources/org/datadog/jenkins/plugins/datadog/DataDogJobProperty/help-tagFile.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    <p>Read a file from workspace as a newline seperated list of tags to push to DataDog. Location is relative to workspace.</p>
+    <p>Example: <em>os=${OS_PARAM}</em>.</p> 
+    <p>The value can either be something that is automatically expanded by Jenkins (environment and/or build variable), or a static value. The OS_PARAM in this case is expanded by Jenkins.</p>
```

Comment:
Let's add a note about the implications of absent tags as well.

`Note: Tags set this way, in the workspace, won't appear in started events. To provide visibility of a jobs lifespan, check the option for sending an extra event after each successful checkout.`